examples/rapids/mlflow_project/src/sample_server_query.sh

```diff
@@ -0,0 +1,2 @@
+curl -X POST -H "Content-Type:application/json; format=pandas-split" --data '{"columns":["Year", "Month", "DayofMonth", "DayofWeek", "CRSDepTime", "CRSArrTime", "UniqueCarrier", "FlightNum", "ActualElapsedTime", "Origin"    , "Dest", "Distance", "Diverted"],"data":[[1987, 10, 1, 4, 1, 556, 0, 190, 247, 202, 162, 1846, 0]]}' http://127.0.0.1:55756/invocations
```

Comment:
Nice, thanks for adding this example input :)

We actually recently added support for logging example input ([see docs](https://mlflow.org/docs/latest/python_api/mlflow.sklearn.html#mlflow.sklearn.log_model)) along with MLflow models in 1.9.0. You can use syntax like the following to log a model with example input:

```
mlflow.sklearn.log_model(model,
    path=path,
    signature=signature,
    input_example=example)
```

And access the example input when loading it back via:

```
model = mlflow.pyfunc.load_model(path)
example_info = model.metadata.saved_input_example_info
example_path = example_info["artifact_path"] # also has fields type -> dataframe, pandas_orient -> split etc
example_df = pd.read_json(example_path, orient = example_info.get("pandas_orient"))
example_prediction = model.predict(example_df)
```

@tomasatdatabricks do you think it'd be worth having such an example / does the above usage pattern look right? We may want to include a section in the MLflow docs explaining how to use the new model schemas/input-examples

Reply:
I'll update the notebook version to use this process. I do think it makes sense to leave in the curl example for the CLI workflow though.